app/assets/stylesheets/pages/_user_config.scss

```diff
@@ -0,0 +1,122 @@
+.user-config {
+  flex-direction: column;
+  overflow: auto;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 5%;
+
+  h2 {
+    font-size: 2rem;
+    text-align: center;
+  }
+
+  h3 {
+    font-size: 1.5rem;
+    text-align: center;
+  }
+
+  .field {
+    flex-direction: column;
+    gap: 4px;
+  
+    label {
+      font-size: 1.5rem;
+    }
+
+    i {
+      font-size: 0.8rem;
+    }
+  
+    input {
+      border-radius: 50px;
+      border: none;
+      padding: 0.5rem 1.5rem;
+      
+      @include themify {
+        background-color: theme('backgroundColor-tertiary');
+        color: theme('colorInverted');
+      }
+    }
+  }
+}
+
+.form .actions {
+  justify-content: space-between;
+  margin-top: 1rem;
+
+  input, button {
+    border-radius: 50px;
+    font-size: 1rem;
+    padding: 0.5rem 3rem;
+    
+    @include themify {
+      background-color: darken(theme('backgroundColor-main'), 10%);
+      border: 1px solid theme('colorInverted');
+      color: theme('colorInverted');
+    }
+  }
+
+  &__warning, &__danger {
+    justify-content: space-between;
+    margin-top: 1rem;
+
+    button {
+      border-radius: 50px;
+      font-size: 1rem;
+      padding: 0.5rem 3rem;
+    }
+
+    button, form {
+      width: 100%;
+    }
+  }
+
+  &__warning button {
+    @include themify {
+      background-color: darken(theme('backgroundColor-main'), 20%);
+      border: 1px solid theme('colorInverted');
+      color: theme('colorInverted');
+    }
+  }
+
+  &__danger button {
+    @include themify {
+      background-color: darken(theme('backgroundColor-tertiary'), 0%);
+      border: 1px solid theme('colorInverted');
+      color: theme('colorInverted');
+    }
+  }
```

Comment:
Isn't there a way to just add a single themify the code repeats a lot

Reply:
i did it, but i couldn't modify the themify in order to do not get error with the others files that use the themify, so i just added another mixin, maybe later in another pr i modify all the files and change the last mixin to the new one

Reply:
Gud